pkg/clients/clients.go

```diff
@@ -174,6 +177,26 @@ func (dcs *DynamicClientSet) IsNamespacedKind(gvk schema.GroupVersionKind) (bool
 	return false, &NoNamespaceInfoErr{gvk}
 }
 
+type LogClient struct {
+	clientset *kubernetes.Clientset
+}
+
+func NewLogClient(clientConfig *rest.Config) (*LogClient, error) {
+	// creates the clientset
+	clientset, err := kubernetes.NewForConfig(clientConfig)
```

Comment:
Any reason you can't reuse the existing clientset here? Is this using the typed client instead?

Reply:
It doesn't support querying, right?